step/src/main/java/com/google/sps/User.java

```diff
@@ -70,7 +70,7 @@ public String getName() {
           preferences = DatabaseHandler.getUserPreferences(userServiceUserId);
       }
       return preferences;
-
+  }
```

Comment:
Um do you know what happened here? Adding a single bracket implies either there was a compile error before or there will be now - either is problematic so I'm confused about this bracket.

Reply:
I am not sure but I think when it had some conflicts with master branch before. The master branch had the bracket and an empty line. And this version had the bracket only which caused conflicts so I deleted and added it again.